### PR TITLE
feat: Local UniswapV3 Markets

### PR DESCRIPTION
**Pull request by davidterpay**

### CODE REVIEW

File: oracle/constants/currency_pairs.go [hunk 1]
```diff
@@ -126,7 +134,25 @@ var (
...
 	ETHEREUM_BITCOIN = pkgtypes.NewCurrencyPair("ETH", "BTC")
+
+	// ETH denominated tickers.
+	AAVE_ETH                            = pkgtypes.NewCurrencyPair("AAVE", "ETH")
+	AXELAR_ETH                          = pkgtypes.NewCurrencyPair("AXL", "ETH")
+	CHAINLINK_ETH                       = pkgtypes.NewCurrencyPair("LINK", "ETH")
+	DAI_ETH                             = pkgtypes.NewCurrencyPair("DAI", "ETH")
+	ETHENA_ETH                          = pkgtypes.NewCurrencyPair("ENA", "ETH")
+	ETHERFI_ETH                         = pkgtypes.NewCurrencyPair("ETHFI", "ETH")
+	HARRY_POTTER_OBAMA_SONIC_10_INU_ETH = pkgtypes.NewCurrencyPair("HARRY_POTTER_OBAMA_SONIC_10_INU", "ETH")
```
Comment (aljo242) [inline oracle/constants/currency_pairs.go:149]: bet

Comment (davidterpay) [reply oracle/constants/currency_pairs.go:149]: kek

**Review by aljo242 — APPROVED** [review]

File: cmd/slinky-config/main.go [hunk 2]
```diff
@@ -302,7 +315,7 @@ func init() {
 		&updateInterval,
 		"update-interval",
 		"",
-		500*time.Millisecond,
+		250*time.Millisecond,
```
Comment (Eric-Warehime) [inline cmd/slinky-config/main.go:318]: This is a relatively small change, but in general I think we should avoid combining unrelated changes into PRs. It makes it more likely for people to miss them.  
Updating the default price aggregation interval has nothing to do with adding uniswap markets.

**Review by Eric-Warehime — APPROVED** [review]

File: cmd/slinky-config/main.go (hunk 2 above)
Comment (davidterpay) [reply cmd/slinky-config/main.go:318]: yea agreed, this was meant to be a fix in previous PRs but got excluded.
